common/src/main/java/org/apache/drill/common/config/NestedConfig.java

```diff
@@ -111,6 +112,19 @@ public String getString(String path) {
     return c.getString(path);
   }
 
+  /**
+   * Returns string option or passed default value if option is not set.
+   *
+   * @param path option path
+   * @param defaultValue default value
+   * @return option or default value
+   */
+  public String getString(String path, String defaultValue) {
```

Comment:
There is no need for this method at all, default values can be set directly in drill-module.conf.